client/src/components/ElementEditor/Header.scss

```diff
@@ -47,10 +47,6 @@
     width: unset;
   }
 
-  &__version-state {
-    box-shadow: $gallery-item-shadow;
-  }
-
```

Comment:
Wasn't this shadow part of the designs?

Reply:
It's removed due to linter taking issue with the equivalent definition immediately below the removed block.

This block also has a `box-shadow`.

If one is more correctorer than the other, it would be great to know which is which. Is @newleeland able to tell from the definitions?

Reply:
Might've been a merge conflict thing, makes sense. Presumably the change you're removing is not used anyway